src/Widgets/FormInput/FormInput.tsx

```diff
@@ -36,9 +38,23 @@ export const FormInput = forwardRef<FormSubmit, FormInputProps>((props, ref):JSX
                
         });
         return formValues;
-   }
+    }
+
+    const validateUserAttribute = async (fieldName:string, fieldValue:any, ):Promise<boolean> => {
+        const fieldQuery = `${fieldName}=${fieldValue}`;
+        try {
+            const response = await axios.get(`${DOMAIN}/resources/available-account?${fieldQuery}`);
+            if (response.status == 204) return true;
+            else return false;
+        } catch (error) {
+            ToastQueueManager.show(error as unknown as AxiosError<ServerErrorResponse>);
+            return false;
+        }
+    }
+        //}).catch((error:AxiosError<ServerErrorResponse>) => {ToastQueueManager.show(error); return false});
+    
```

Comment:
Thanks Josh.  Once these lines are deleted, I think we're good to wrap this one up.